src/Standards/PSR12/Sniffs/ControlStructures/BooleanOperatorPlacementSniff.php

```diff
@@ -67,7 +75,7 @@ public function process(File $phpcsFile, $stackPtr)
         ];
 
         $operator  = $parenOpener;
-        $position  = null;
+        $position  = $this->getDefaultPosition();
```

Comment:
I don't think a private function is required here. I can't see this code being reused in the sniff.

Reply:
Done